DataConnectors/WireXSystems/sample/wirex_sample data.csv

```diff
@@ -0,0 +1,224 @@
+﻿TenantId,SourceSystem,TimeGenerated,ReceiptTime,DeviceVendor,DeviceProduct,DeviceEventClassID,LogSeverity,OriginalLogSeverity,DeviceAction,SimplifiedDeviceAction,Computer,CommunicationDirection,DeviceFacility,DestinationPort,DestinationIP,DeviceAddress,DeviceName,Message,Protocol,SourcePort,SourceIP,RemoteIP,RemotePort,MaliciousIP,ThreatSeverity,IndicatorThreatType,ThreatDescription,ThreatConfidence,ReportReferenceLink,MaliciousIPLongitude,MaliciousIPLatitude,MaliciousIPCountry,DeviceVersion,Activity,ApplicationProtocol,EventCount,DestinationDnsDomain,DestinationServiceName,DestinationTranslatedAddress,DestinationTranslatedPort,DeviceDnsDomain,DeviceExternalID,DeviceInboundInterface,DeviceNtDomain,DeviceOutboundInterface,DevicePayloadId,ProcessName,DeviceTranslatedAddress,DestinationHostName,DestinationMACAddress,DestinationNTDomain,DestinationProcessId,DestinationUserPrivileges,DestinationProcessName,DeviceTimeZone,DestinationUserID,DestinationUserName,DeviceMacAddress,ProcessID,ExternalID,FileCreateTime,FileHash,FileID,FileModificationTime,FilePath,FilePermission,FileType,FileName,FileSize,ReceivedBytes,OldFileCreateTime,OldFileHash,OldFileID,OldFileModificationTime,OldFileName,OldFilePath,OldFilePermission,OldFileSize,OldFileType,SentBytes,RequestURL,RequestClientApplication,RequestContext,RequestCookies,RequestMethod,SourceHostName,SourceMACAddress,SourceNTDomain,SourceDnsDomain,SourceServiceName,SourceTranslatedAddress,SourceTranslatedPort,SourceProcessId,SourceUserPrivileges,SourceProcessName,SourceUserID,SourceUserName,EventType,DeviceCustomIPv6Address1,DeviceCustomIPv6Address1Label,DeviceCustomIPv6Address2,DeviceCustomIPv6Address2Label,DeviceCustomIPv6Address3,DeviceCustomIPv6Address3Label,DeviceCustomIPv6Address4,DeviceCustomIPv6Address4Label,DeviceCustomFloatingPoint1,DeviceCustomFloatingPoint1Label,DeviceCustomFloatingPoint2,DeviceCustomFloatingPoint2Label,DeviceCustomFloatingPoint3,DeviceCustomFloatingPoint3Label,DeviceCustomFloatingPoint4,DeviceCustomFloatingPoint4Label,DeviceCustomNumber1,DeviceCustomNumber1Label,DeviceCustomNumber2,DeviceCustomNumber2Label,DeviceCustomNumber3,DeviceCustomNumber3Label,DeviceCustomString1,DeviceCustomString1Label,DeviceCustomString2,DeviceCustomString2Label,DeviceCustomString3,DeviceCustomString3Label,DeviceCustomString4,DeviceCustomString4Label,DeviceCustomString5,DeviceCustomString5Label,DeviceCustomString6,DeviceCustomString6Label,DeviceCustomDate1,DeviceCustomDate1Label,DeviceCustomDate2,DeviceCustomDate2Label,FlexDate1,FlexDate1Label,FlexNumber1,FlexNumber1Label,FlexNumber2,FlexNumber2Label,FlexString1,FlexString1Label,FlexString2,FlexString2Label,AdditionalExtensions,StartTime,EndTime,Type,"_ResourceId"
```

Comment:
Feedback on the sample data file:
1. Please move the sample data file to https://github.com/Azure/Azure-Sentinel/tree/master/Sample%20Data/CEF folder since this is a CEF data connector sample data.
2. I see just one example of DeviceAction. Can you please cover all combinations of DeviceAction that your product supports and relevant fields that gets populated depending on the different types of DeviceAction in the sample data?
3. Same feedback as point 2 above for Simplified Device action
4. On all the fields which are not populated like "communication Direction", etc. are these always not populated for any type of logs your product provides? If yes, that's fine. If there are some fields that do get populated for specific log types, please provide that sample data filling those fields and relevant applicable properties as well. 
5. For those fields where Device Action is specified, why isn't Device address and Device name fields populated?
6. Since these are forensic logs, wouldn't the malicious IP, malicious IP geo location, file information and threat information fields not captured in the original source logs on WireX side?
7. I see the additional extensions have lots of valuable info like host address, proc address, Db information, record type etc. fields. Is it possible to write a parser to extract these field values and populate relevant columns so that customers can easily correlate this data? Examples of parsers are at https://github.com/Azure/Azure-Sentinel/tree/master/Parsers. Specific example of a similar parser built for CEF data source parsing is https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/TrendMicro/TrendMicroDeepSecurity

Reply:
1)I am going to collect move sample and then do as instructed, thank you
2)There is no Device action
3)same as 2
4) not populated
5)There is not action 
6)these are all logs for forensics so there is no determination of malicious or not
7) I agree that this would be a better long term solution but for now we need to get a data connector in and functioning. and then over time make it better.